test/suite/controller/addonController.test.ts

```diff
@@ -129,6 +129,12 @@ describe("addonController.ts", async () => {
     }
   });
 
+  after(async () => {
+    if (fs.existsSync(extractedVersionFolder)) {
```

Comment:
I am wondering why we remove the files after _all_ tests have been executed, and not after every test in `afterEach()`?

Reply:
That's what I originally went with, but much of this test code is a holdover from the original Assay codebase and (seemingly) depend on one another. This felt safest.